src/Sylius/Behat/Element/Product/ShowPage/PricingElement.php

```diff
@@ -22,6 +22,10 @@ public function getPriceForChannel(string $channelName): string
         /** @var NodeElement $priceForChannel */
         $channelPriceRow = $this->getDocument()->find('css', sprintf('#pricing tr:contains("%s")', $channelName));
 
+        if (null === $channelPriceRow) {
+            return '';
```

Comment:
Maybe `null`? It clearly says that *there is no price* rather than *there is something but empty* 💃